src/JSInterop/Microsoft.JSInterop/src/DotNetObjectRef.cs

```diff
@@ -15,7 +15,8 @@ public static class DotNetObjectRef
         /// <returns>An instance of <see cref="DotNetObjectRef{TValue}" />.</returns>
         public static DotNetObjectRef<TValue> Create<TValue>(TValue value) where TValue : class
         {
-            return new DotNetObjectRef<TValue>(value);
+            var objectId = DotNetObjectRefManager.Current.TrackObject(value);
```

Comment:
We were doing this in the ctor of `DotNetObjectRef<T>` in the past. However, we don't actually need the ref instance to be cached - it's never used. This just makes it much cleaner.